### DISCUSSION TITLE
I don't  know what happened at production no style is applied as intend

### DISCUSSION THREAD

**Discussion by SahasSaurav** [vercel/next.js#20603] [Help]
At development my app looks like this
![Screenshot (92)](https://user-images.githubusercontent.com/60291223/103354184-e4f75700-4ad0-11eb-8004-f7fe80d11ffc.png)

but at it looks like this i dont knw what happened at production
![Screenshot (93)](https://user-images.githubusercontent.com/60291223/103354213-fb9dae00-4ad0-11eb-9eec-ea930e3a0da7.png)

link for repo
https://github.com/SahasSaurav/next_pomodoro 

i m using context api along with usereducer ,react, next, typescript and tailwindcss

**Comment by awareness481** [comment]
It's probably an incorrect setup with purging.

In your [tailwind.config.js](https://github.com/SahasSaurav/next_pomodoro/blob/main/tailwind.config.js) file try changing the `purge` key to

`purge: ["./src/**/*.html", "./src/**/*.js"],`
(didn't try it locally but it should work)
